components/eventing-controller/README.md

```diff
@@ -64,8 +64,7 @@ This section explains how to use the Eventing Controller. It expects the followi
 | `DEFAULT_MAX_IN_FLIGHT_MESSAGES`  | The maximum idle "in-flight messages" sent by NATS to the sink without waiting for a response. |
 | `DEFAULT_DISPATCHER_RETRY_PERIOD` | The retry period for resending an event to a sink, if the sink doesn't return 2XX.             |
 | `DEFAULT_DISPATCHER_MAX_RETRIES`  | The maximum number of retries to send an event to a sink in case of errors.                    |
-| **For NATS JetStream**            |                                                                                                |
-|  `JS_STREAM_NAME`                 | Name of the stream where all events are stored.                                                |
+| **For NATS JetStream**            |                                                                                                | |
```

Comment:
There's an extra `|` at the end.

Reply:
It's not extra, its there to make an empty column in-front of  `For NATS JetStream`


![image](https://user-images.githubusercontent.com/9897945/160613297-3ca75f96-c7d2-4cb6-8853-f6ba387db18c.png)

Reply:
removed